lib/shoulda/matchers/active_record/association_matcher.rb

```diff
@@ -1292,7 +1299,8 @@ def class_name_correct?
         end
 
         def join_table_correct?
-          if macro != :has_and_belongs_to_many || join_table_matcher.matches?(@subject)
+          if macro != :has_and_belongs_to_many || \
```

Comment:
When it comes to stretching if statements across multiple lines, what do you think about formatting this as:

``` ruby
if (
  macro != :has_and_belongs_to_many ||
  join_table_matcher.matches?(@subject)
)
```